test/support/api_spec.ex

```diff
@@ -1,4 +1,6 @@
 defmodule OpenApiSpexTest.ApiSpec do
+  @moduledoc false
```

Comment:
Why add this attribute to a test support file? If this is from a credo check then I suggest we disable it.